code/go/0chain.net/smartcontract/faucetsc/sc.go

```diff
@@ -154,6 +154,12 @@ func toSeconds(dur time.Duration) common.Timestamp {
 func (fc *FaucetSmartContract) pour(t *transaction.Transaction, _ []byte, balances c_state.StateContextI, gn *GlobalNode) (string, error) {
 	user := fc.getUserVariables(t, gn, balances)
 	ok, err := user.validPourRequest(t, balances, gn)
+	if err != nil {
+		logging.Logger.Info("pour_error: error while validating pour request",
+			zap.String("transaction_hash", t.Hash),
+			zap.Error(err))
+		return "", common.NewErrorf("pour_failed", "error validating pour request: %v", err)
```

Comment:
SC errors will all be logged at the top level when the txn is executed, we should avoid duplicate logging, which will make the log files size increasing fast and flush useful logs perhaps